pkg/apis/kubeflow/v2beta1/types.go

```diff
@@ -54,11 +54,35 @@ const (
 // SchedulingPolicy encapsulates various scheduling policies of the distributed training
 // job, for example `minAvailable` for gang-scheduling.
 type SchedulingPolicy struct {
-	MinAvailable           *int32           `json:"minAvailable,omitempty"`
-	Queue                  string           `json:"queue,omitempty"`
-	MinResources           *v1.ResourceList `json:"minResources,omitempty"`
-	PriorityClass          string           `json:"priorityClass,omitempty"`
-	ScheduleTimeoutSeconds *int32           `json:"scheduleTimeoutSeconds,omitempty"`
+	// MinAvailable defines the minimal number of member to run the PodGroup.
+	// If the gang-scheduling is set to the volcano,
+	// input is passed to `.spec.mimMember` in PodGroup for the volcano.
```

Comment:
Explain that this will only work if the MPI application supports resizing.

Reply:
Sounds good.

Reply:
Done.

Reply:
If not set, it defaults to the number of workers